l10n_ch_import_camt054/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+{
+    'name': 'CAMT 054 import and reconcile',
+    'version': '10.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'Monzione Marco',
+    'website': '...',
+    'category': 'Banking addons',
+    'depends': [
+        'account_bank_statement_import_camt',
```

Comment:
camt_details is enough because it already depends on account_bank_statement_import_camt. You can remove this line